runners/flink/job-server/flink_job_server.gradle

```diff
@@ -155,3 +156,24 @@ task validatesPortableRunner() {
   dependsOn validatesPortableRunnerBatch
   dependsOn validatesPortableRunnerStreaming
 }
+
+project.ext.validatesCrossLanguageTransforms =
+  createPortableValidatesRunnerTask(
```

Comment:
`validatesCrossLanguageTransforms` task runs `ExternalTest`. It will be replaced with `validateCrossLanguageRunner` task when #8174 is done but I added it here for the completeness of this PR.